src/sound/xactwavebank_ascii.cpp

```diff
@@ -63,7 +63,11 @@ RewindableAudioStream *XACTWaveBank_ASCII::getWave(size_t index) const {
 	if (!dataStream)
 		throw Common::Exception("XACTWaveBank_ASCII::getWave(): No such resource \"%s\"", wave.name.c_str());
 
+#ifdef ENABLE_VORBIS
 	return makeVorbisStream(dataStream.release(), true);
+#else
+	return 0;
```

Comment:
It should even throw, with a message like https://github.com/xoreos/xoreos/blob/master/src/sound/sound.cpp#L295